examples/images/voc_sift_fisher.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+#Set environment variables
+: ${KEYSTONE_MEM:=4g}
+export KEYSTONE_MEM
+
+KEYSTONE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/../..
+: ${HADOOP_EXAMPLE_DATA_DIR:=/example_data}
+
+
+#Get the data and copy to HDFS if necessary.
+if [ ! hadoop fs -test -e $HADOOP_EXAMPLE_DATA_DIR/VOCtrainval_06-Nov-2007.tar ]; then
+    wget -O $TMPDIR/VOCtrainval_06-Nov-2007.tar https://s3-us-west-2.amazonaws.com/voc-data/VOCtrainval_06-Nov-2007.tar
+    hadoop fs -copyFromLocal VOCtrainval_06-Nov-2007.tar $HADOOP_EXAMPLE_DATA_DIR
```

Comment:
Yeah, for now. The TIMIT and Amazon datasets require special agreements to redistribute, so we can't just make them publicly available. Honestly the input data for VOC is small enough that it might be fine to refactor such that it's on the master and then `sc.parallelize()` the training data.